spec/features/comment_on_a_pic_spec.rb

```diff
@@ -0,0 +1,14 @@
+# frozen_string_literal: true
+
+require 'rails_helper'
+
+RSpec.describe "Commenting", type: :feature do
+  it "Users can comment" do
+    visit "/pics/new"
+    attach_file("Image", Rails.root + "spec/fixtures/files/happy-face.png")
```

Comment:
Helper for this? (Per review: https://stackoverflow.com/questions/3466083/imperative-and-declarative-steps-in-rspec)